### PR TITLE
arabic translation

### PR DESCRIPTION
**Pull request by atefBB**

### CODE REVIEW

File: public/ar-ar/index.html [hunk 1]
```diff
@@ -151,7 +151,7 @@
 <h3>النسخة الحالية</h3>
 
 <p>
-    معيار جافاسكريبت هي إيكماسكريبت (ECMAScript). اعتبارا من 2012، كل المتصفحات الحديثة تدعم بشكل كامل  إيكماسكريبت 5.1. المتصفحات القديمة تدعم على الأقل إيكماسكريبت 3. اعتبارا من 2015 تمت الموافقة على مواصفات ES6/ES2015. بعد دورة الإصدار الجديدة، تم اعتماد ES7/ES2016 في شهر يونيو/جزيران (جوان أو الشهر السادس من شهور السنة الشمسية) 2016. ألق نظرة على مواصفة لغة إيكماسكريبت 2016 على <a target="_blank" href="http://www.ecma-international.org/ecma-262/7.0/index.html">Ecma International</a>.
```
Comment (allanesquina) [inline public/ar-ar/index.html]: Hey @atefBB, make sure you're changing the right files. The files within public folder are auto generated from partials templates. 
https://github.com/braziljs/js-the-right-way/tree/master/i18n/ar-ar/partials
:+1:

Comment (atefBB) [inline public/ar-ar/index.html]: Hey ! you mean I should not update `~public/ar-ar/index.html` ?

Comment (allanesquina) [inline public/ar-ar/index.html]: Yup, these files are generated automatically.
In this case, you should change that file:
https://github.com/braziljs/js-the-right-way/blob/master/i18n/ar-ar/partials/getting-started.html

Comment (atefBB) [inline public/ar-ar/index.html]: Aha ! So I'll fix it ! thx.

Comment (atefBB) [inline public/ar-ar/index.html]: Fixed https://github.com/braziljs/js-the-right-way/pull/304/commits/258f5f22c942d80675fc98b68a30ad0d7d449285

**Comment by allanesquina** [comment]
Hey @atefBB, can I merge that?

**Comment by atefBB** [comment]
YES plz !

**Comment by allanesquina** [comment]
Thanks @atefBB and welcome :+1:

**Pull request merged**